src/blockchains/filecoin.ts

```diff
@@ -0,0 +1,73 @@
+import {BlockchainHandler, BlockchainHandlerOpts} from "../blockchain-handler";
+import {AccountID} from "caip";
+import {ConsentMessage, getConsentMessage, LinkProof} from "../utils";
+import type { MessageParams } from '@zondax/filecoin-signing-tools'
+
+const namespace = 'fil'
+
+const moduleToImport = process.env.JEST_WORKER_ID ? "@zondax/filecoin-signing-tools/nodejs" : "@zondax/filecoin-signing-tools"
+
+function asTransaction(address: string, message: string): MessageParams {
```

Comment:
Oh, there is no way to just sign a raw message?

Reply:
This is not provided by the Filecoin signing tools. I believe it is not a big deal, as only constraint here is signed payload has predictable form, and `verify(sign(payload)) == true` constraint is valid.